src/facets/PantosRegistryFacet.sol

```diff
@@ -815,7 +822,7 @@ contract PantosRegistryFacet is IPantosRegistry, PantosBaseFacet {
     function _registerBlockchain(
         uint256 blockchainId,
         string memory name
-    ) private onlyOwner {
+    ) private onlyRole(SUPER_CRITICAL_OPS) {
```

Comment:
registerBlockchain is already using `onlyRole(SUPER_CRITICAL_OPS)`. Do we need another one here ?

Reply:
I also don't think we need it. I just kept the old structure. I will keep it like this for now, but I will keep a note to discuss it with @markuslevonyak when he's back. Maybe there was a reason for this to be added and I am overlooking it. Better safe than sorry :D